blocks/extra-large-manual-promo-block/_index.scss

```diff
@@ -3,16 +3,4 @@
 .b-xl-manual-promo {
 	@include scss.block-properties("xl-manual-promo");
 	@include scss.block-components("xl-manual-promo");
-	&__overline {
-		@include scss.block-properties("xl-manual-promo-overline-hover");
-		@include scss.block-components("xl-manual-promo-overline-hover");
-	}
-	&__headline {
-		@include scss.block-properties("xl-manual-promo-headline-hover");
-		@include scss.block-components("xl-manual-promo-headline-hover");
-	}
-	&__img {
-		@include scss.block-properties("xl-manual-promo-img");
-		@include scss.block-components("xl-manual-promo-img");
-	}
```

Comment:
Keep the image token so we can apply the opacity to the correct element.

Reply:
@vgalatro Fixed it in [a622ba2](https://github.com/WPMedia/arc-themes-blocks/pull/1690/commits/a622ba26042fa29d7ded26bda4f557df321862c0)